tests/e2e/configurer/chain/commands.go

```diff
@@ -35,10 +38,40 @@ func (n *NodeConfig) CreatePool(poolFile, from string) uint64 {
 	err = util.Cdc.UnmarshalJSON(bz, &numPools)
 	require.NoError(n.t, err)
 	poolID := numPools.NumPools
-	n.LogActionF("successfully created pool %d", poolID)
+	n.LogActionF("successfully created balancer pool %d", poolID)
 	return poolID
 }
 
+func (n *NodeConfig) CreateConcentratedPool(from string) poolmanagertypes.PoolI {
+	n.LogActionF("creating concentrated pool")
+
+	var (
+		denom1             = "uosmo"
+		denom2             = "gamm/pool/1"
```

Comment:
I don't understand why we are using gamm shares as denom2, can we use something more realistic?